docker-compose.yml

```diff
@@ -81,11 +87,25 @@ services:
       CELERY_RESULT_BACKEND: redis://redis:6379/4
       ELASTICSEARCH_URL: elastic:9200
       CELERY_ALWAYS_EAGER: 'False'
+      EXAMS_SFTP_HOST: sftp
+      EXAMS_SFTP_USERNAME: odl
+      EXAMS_SFTP_PASSWORD: 123
+      EXAMS_SFTP_UPLOAD_DIR: share
     env_file: .env
     links:
       - db
       - elastic
       - redis
+      - sftp
+
+  sftp:
+    image: atmoz/sftp
+    volumes:
+        - sftp_share:/home/odl/share
+    ports:
+        - "2022:22"
+    command: odl:123:1001:1001:share
```

Comment:
Unix user/group id for the `odl` user getting created here.  The argument syntax the author chose is `user:pass[:uid[:gid[:dir]...]]`, all I really wanted was `share`, so I had to specify all of the args leading up to it.